CHANGELOG.md

```diff
@@ -4,8 +4,36 @@
 Darts is still in an early development phase and we cannot always guarantee backwards compatibility. Changes that may **break code which uses a previous release of Darts** are marked with a "&#x1F534;".
 
 ## [Unreleased](https://github.com/unit8co/darts/tree/develop)
-[Full Changelog](https://github.com/unit8co/darts/compare/0.8.1...develop)
+[Full Changelog](https://github.com/unit8co/darts/compare/0.9.0...develop)
 
+## [0.9.0](https://github.com/unit8co/darts/tree/0.9.0) (2021-07-09)
+### For users of the library:
+
+**Added:**
+- Multiple forecasting models can now produce probabilistic forecasts by specifying a `num_samples` parameter when calling `predict()`. Stochastic forecasts are stored by utilizing the new `samples` dimension in the refactored `TimeSeries` class (see 'Changed' section). Models supporting probabilistic predictions so far are `ARIMA`, `ExponentialSmoothing`, `RNNModel` and `TCNModel`.
+- Introduced `LikelihoodModel` class which is used by probabilistic `TorchForecastingModel` classes in order to make predictions in the form of parametrized distributions of different types.
+- Added new abstract class `TorchParametricProbabilisticForecastingModel` to serve as parent class for probabilistic models.
+- Introduced new `FilteringModel` abstract class alongside `MovingAverage`, `KalmanFilter` and `GaussianProcessFilter` as concrete implementations.
+- Future covariates are now utilized by `TorchForecastingModels` when the forecasting horizon exceeds the `output_chunk_length` of the model. Before, `TorchForecastingModel` instances could only predict beyond their `output_chunk_length` if they were not trained on covariates, i.e. if they predicted all the data they need as input. This restriction has now been lifted by letting a model not only consume its own output when producing long predictions, but also utilizing the covariates known in the future, if available.
+- Added a new `RNNModel` class which utilizes and rnn module as both encoder and decoder. This new class natively supports the use of the most recent future covariates when making a forecast. See documentation for more details.
+- Introduced optional `epochs` parameter to `TorchForecastingModel` instances which, if provided, overrides the `n_epochs` attribute in that particular training session.
```

Comment:
```suggestion
- Introduced optional `epochs` parameter to `TorchForecastingModel.predict()` method, which, if provided, overrides the `n_epochs` attribute in that particular model instance.
```

Reply:
Maybe we should say '...in that particular model instance and training session.'? Because I'm afraid otherwise it might suggest that the model attribute `n_epochs` itself is replaced. What do you think?

Reply:
Sounds good, agree!